docker-compose-gpu.yml

```diff
@@ -22,19 +23,22 @@ services:
     environment:
       # See rest_api/pipelines.yaml for configurations of Search & Indexing Pipeline.
       - DOCUMENTSTORE_PARAMS_HOST=elasticsearch
+      - PIPELINE_YAML_PATH=/home/user/rest_api/pipeline/pipeline_dpr.yaml
     depends_on:
       - elasticsearch
     command: "/bin/bash -c 'sleep 10 && gunicorn rest_api.application:app -b 0.0.0.0 -k uvicorn.workers.UvicornWorker --workers 1 --timeout 180'"
+
   elasticsearch:
     # This will start an empty elasticsearch instance (so you have to add your documents yourself)
     #image: "elasticsearch:7.9.2"
     # If you want a demo image instead that is "ready-to-query" with some indexed Game of Thrones articles:
-    image: "deepset/elasticsearch-game-of-thrones"
+    image: "julianrisch/elasticsearch-countries-and-capitals"
```

Comment:
let's make this a deepset image

Reply:
Tested with the new deepset image, works well!